packages/mdc-tab-indicator/_mixins.scss

```diff
@@ -0,0 +1,64 @@
+/**
+  * @license
+  * Copyright 2018 Google Inc. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License")
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+@import "@material/theme/mixins";
+
+@mixin mdc-tab-indicator-active {
+  opacity: 1;
+}
+
+@mixin mdc-tab-indicator-color($color) {
+  @include mdc-tab-indicator-bar-color($color);
+  @include mdc-tab-indicator-icon-color($color);
+}
+
+@mixin mdc-tab-indicator-height($height) {
```

Comment:
this mixin will probably end up outputting more CSS than the client needs. I would recommend we only support `mdc-tab-indicator-bar-height` and `mdc-tab-indicator-icon-height`.